### PR TITLE
Update legal and support help texts

### PR DESCRIPTION
**Pull request by jasonblais**
#### Summary
Update legal and support help texts

#### Ticket Link
N/A

#### Checklist
- [X] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates

### CODE REVIEW

**Comment by jasonblais** [comment]
maybe @JeffSchering you can help review the changes here?

File: webapp/components/admin_console/legal_and_support_settings.jsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.privacyDesc'
-                            defaultMessage='Link to Privacy Policy available to users on desktop and on mobile. Leaving this blank will hide the option to display a notice.'
+                            defaultMessage='Link to Privacy Policy available to users on desktop and on mobile. Leaving this field blank will hide the option to display a notice.'
```
Comment (JeffSchering) [inline webapp/components/admin_console/legal_and_support_settings.jsx]: The URL for the Privacy link on the login and sign-up pages. If this field is empty, the Privacy link is hidden from users.

File: webapp/components/admin_console/legal_and_support_settings.jsx [hunk 2]
```diff
@@ -96,7 +96,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.aboutDesc'
-                            defaultMessage='Link to About page for more information on your Mattermost deployment, for example its purpose and audience within your organization. Defaults to Mattermost information page.'
+                            defaultMessage='Link to About page for more information on your Mattermost deployment, for example its purpose and audience within your organization. Leave this field blank to hide the link in the login and sign-up pages.'
```
Comment (JeffSchering) [inline webapp/components/admin_console/legal_and_support_settings.jsx]: The URL for the About link on the Mattermost login and sign-up pages. If this field is empty, the About link is hidden from users.

File: webapp/components/admin_console/legal_and_support_settings.jsx [hunk 3]
```diff
@@ -113,7 +113,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.helpDesc'
-                            defaultMessage='Link to help documentation from team site main menu. Typically not changed unless your organization chooses to create custom documentation.'
+                            defaultMessage='Link to help documentation from the Main Menu. Typically not changed unless your organization chooses to create custom documentation. Leave this field blank to hide the option from the Main Menu.'
```
Comment (JeffSchering) [inline webapp/components/admin_console/legal_and_support_settings.jsx]: The URL for the Help link on the Mattermost login page, sign-up pages, and Main Menu. If this field is empty, the Help link is hidden from users.

File: webapp/components/admin_console/legal_and_support_settings.jsx [hunk 4]
```diff
@@ -130,7 +130,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.problemDesc'
-                            defaultMessage='Link to help documentation from team site main menu. By default this points to the peer-to-peer troubleshooting forum where users can search for, find and request help with technical issues.'
+                            defaultMessage='Link to help documentation from the Main Menu. By default this points to the peer-to-peer troubleshooting forum where users can search for, find and request help with technical issues. Leave this field blank to hide the option from the Main Menu.'
```
Comment (JeffSchering) [inline webapp/components/admin_console/legal_and_support_settings.jsx]: The URL for the Report a Problem link in the Main Menu. If this field is empty, the link is removed from the Main Menu.

File: webapp/i18n/en.json [hunk 5]
```diff
@@ -828,17 +828,17 @@
   "admin.sql.traceDescription": "(Development Mode) When true, executing SQL statements are written to the log.",
   "admin.sql.traceTitle": "Trace: ",
   "admin.sql.warning": "Warning: regenerating this salt may cause some columns in the database to return empty results.",
-  "admin.support.aboutDesc": "Link to About page for more information on your Mattermost deployment, for example its purpose and audience within your organization. Defaults to Mattermost information page.",
+  "admin.support.aboutDesc": "Link to About page for more information on your Mattermost deployment, for example its purpose and audience within your organization. Leave this field blank to hide the link in the login and sign-up pages.",
```
Comment (JeffSchering) [inline webapp/i18n/en.json]: See previous comment.

File: webapp/i18n/en.json [hunk 6]
```diff
@@ -828,17 +828,17 @@
...
   "admin.support.aboutTitle": "About link:",
   "admin.support.emailHelp": "Email address displayed on email notifications and during tutorial for end users to ask support questions.",
   "admin.support.emailTitle": "Support Email:",
-  "admin.support.helpDesc": "Link to help documentation from team site main menu. Typically not changed unless your organization chooses to create custom documentation.",
+  "admin.support.helpDesc": "Link to help documentation from the Main Menu. Typically not changed unless your organization chooses to create custom documentation. Leave this field blank to hide the option from the Main Menu.",
```
Comment (JeffSchering) [inline webapp/i18n/en.json]: See previous comment.

File: webapp/i18n/en.json [hunk 7]
```diff
@@ -828,17 +828,17 @@
...
   "admin.support.helpTitle": "Help link:",
   "admin.support.noteDescription": "If linking to an external site, URLs should begin with http:// or https://.",
   "admin.support.noteTitle": "Note:",
-  "admin.support.privacyDesc": "Link to Privacy Policy available to users on desktop and on mobile. Leaving this blank will hide the option to display a notice.",
+  "admin.support.privacyDesc": "Link to Privacy Policy available to users on desktop and on mobile. Leaving this field blank will hide the option to display a notice.",
```
Comment (JeffSchering) [inline webapp/i18n/en.json]: See previous comment.

File: webapp/i18n/en.json [hunk 8]
```diff
@@ -828,17 +828,17 @@
...
   "admin.support.privacyTitle": "Privacy Policy link:",
-  "admin.support.problemDesc": "Link to help documentation from team site main menu. By default this points to the peer-to-peer troubleshooting forum where users can search for, find and request help with technical issues.",
+  "admin.support.problemDesc": "Link to help documentation from the Main Menu. By default this points to the peer-to-peer troubleshooting forum where users can search for, find and request help with technical issues. Leave this field blank to hide the option from the Main Menu.",
```
Comment (JeffSchering) [inline webapp/i18n/en.json]: See previous comment.

**Comment by jasonblais** [comment]
Thanks! @JeffSchering updated

**Pull request merged**